hack/test-cmd.sh

```diff
@@ -682,9 +682,7 @@ __EOF__
 
   ### Perform a rolling update with --image
   # Command
-  kubectl rolling-update frontend --image=kubernetes/pause --update-period=10ns --poll-interval=10ms "${kube_flags[@]}"
-  # Post-condition: current image IS kubernetes/pause
-  kube::test::get_object_assert 'rc frontend' '{{range \$c:=$rc_container_image_field}} {{\$c.image}} {{end}}' ' +kubernetes/pause +'
+  [ ! kubectl rolling-update frontend --image=kubernetes/pause --timeout=1s --update-period=10ns --poll-interval=10ms "${kube_flags[@]}" || true
```

Comment:
We can add an e2e test case, where we do rollingupdate with an invalid image and see that it fails. And then rollback clears up things.